src/cameras.jl

```diff
@@ -1,200 +1,175 @@
+"""
+Representation of camera parameters such as position, rotation and zoom level.
+
+ - `camera_center::VecE2`: position of camera in [N,E] relative to the mean point. meters
+ - `camera_zoom::Float64`: camera zoom in [pix/m]
+ - `camera_rotation::Float64`: camera rotation in [rad]
+"""
+@with_kw mutable struct CameraState
+    position  :: VecE2 = VecE2(0.,0.)
+    zoom      :: Float64 = 1.
+    rotation  :: Float64 = 0.
+    canvas_width  :: Int64 = DEFAULT_CANVAS_WIDTH
+    canvas_height :: Int64 = DEFAULT_CANVAS_HEIGHT
+end
+position(cs::CameraState) = cs.position
+zoom(cs::CameraState) = cs.zoom
+rotation(cs::CameraState) = cs.rotation
+canvas_width(cs::CameraState) = cs.canvas_width
+canvas_height(cs::CameraState) = cs.canvas_height
```

Comment:
not sure if "canvas" is the best name here, suggestions are welcome

Reply:
seems fine to me, and consistent with the Cairo naming